### DISCUSSION TITLE
Is Hot Reloading Not Supported for Textures?

### DISCUSSION THREAD

**Discussion by zicklag** [bevyengine/bevy#1413] [Q&A] [answered]
I can't seem to get images to hot-reload in Bevy. For instance, if I just add `asset_server.watch_for_chages()` to the Bevy sprite example and then change the sprite while the game is running.

Is this a known issue, future plan, or a bug?

**Comment by tigregalis** [comment]
I wonder if it could be related to https://github.com/bevyengine/bevy/issues/1243 (in that case, the texture is being updated by the CPU)

**Comment by alice-i-cecile** [comment]
I expect that this is a bug; could you file an issue?

**Reply by zicklag** [reply]
I think that this issue might already be it: https://github.com/bevyengine/bevy/issues/1161. So we probably don't need a new one?

**Reply by alice-i-cecile** [reply]
Ah, got it :) I agree now that I've taken a closer look.

**Comment by zicklag** [comment, marked as answer]
Looks like #1161 is fixed now. :tada:
